pkg/ingest/setup.go

```diff
@@ -139,3 +142,23 @@ func setContainerPrefix(prefix *string, isRelease bool, config *v1.BuildConfig)
 	}
 	return nil
 }
+
+func setHelmRepository(repository *string, isRelease bool, config *v1.BuildConfig) error {
+	if config.ReleaseHelmRepository == nil {
+		return fmt.Errorf("must provide a release_helm_repository")
+	}
+	if config.TestHelmRepository == nil {
+		return fmt.Errorf("must provide a test_helm_repository")
```

Comment:
first thought is that this should be consistent with the `setContainerPrefix` and fall back on the `ReleaseHelmRepository`
second thought: not all projects are producing helm resources

suggestion: fallback on the release repository, introduce a new helm repo type called "None" or similar

what do you think?

Reply:
since I think eventually, we want all projects to produce helm resources and in order to have a consistent generic makefile it would be reasonable to require that project that do not use Helm should specify a dummy value